test/adapter/json/nested_serializers_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
+      class Json
+        class NestedSerializersTest < Minitest::Test
+          def setup
+            @tweet = Tweet.new(id: 1, body: 'Tweet 1', date: 'Jan 15')
+            @share1 = Share.new(id: 1, platform: 'facebook', date: 'Jan 16')
+            @author = Author.new(id: 1, name: 'Lucas H.')
```

Comment:
True, that's why I opened #1206. I really think serializers/resources should be defined in each test file somehow.